wearApp/src/main/java/dev/johnoreilly/confetti/wear/speakerdetails/SpeakerDetailsViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package dev.johnoreilly.confetti.wear.speakerdetails
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dev.johnoreilly.confetti.ConfettiRepository
+import dev.johnoreilly.confetti.fragment.SpeakerDetails
+import dev.johnoreilly.confetti.wear.speakerdetails.navigation.SpeakerDetailsDestination
+import kotlinx.coroutines.flow.*
+
+
+class SpeakerDetailsViewModel(
+    savedStateHandle: SavedStateHandle,
+    repository: ConfettiRepository
+) : ViewModel() {
+
+    private val speakerId: String? = savedStateHandle[SpeakerDetailsDestination.speakerIdArg]
+
+    val speaker: StateFlow<SpeakerDetails?> = repository.speakers.map {
+        it.firstOrNull { it.id == speakerId }
+    }.stateIn(viewModelScope, SharingStarted.WhileSubscribed(5000), null)
```

Comment:
Heads up I'm going to remove `speakers` cause it relies on the statefulness of `ConfettiRepository` but I can merge later.

Also note to myself I should add `speaker(id)` in the backend